lib/matplotlib/texmanager.py

```diff
@@ -146,6 +154,8 @@ def __init__(self):
         self._font_preamble = '\n'.join(
             [r'\usepackage{type1cm}'] + cmd + [r'\usepackage{textcomp}'])
 
+        return self
```

Comment:
Why is `_reinit` returning `self`?  I don't see the return value being used anywhere.

Reply:
Leftover of when I tried to put everything in `__new__`... sorry for the mess.